docs/getting-started/first-service.md

```diff
@@ -28,7 +28,8 @@ For those of you familiar with other **source-to-url** tools, this may seem fami
     spec:
       template:
         metadata:
-          name: world
+# This is the name of our new "Revision," see "?" box below for more details
```

Comment:
Seems this change makes this look awkward on the website

Reply:
Just noticed, should have used the netlist preview. 

Maybe the comment needs to be indented

Reply:
@csantanapr opened up #3661 for this